backend/btrixcloud/colls.py

```diff
@@ -217,34 +278,46 @@ async def list_collections(
         except (IndexError, ValueError):
             total = 0
 
-        collections = [Collection.from_dict(res) for res in items]
+        collections = [CollOut.from_dict(res) for res in items]
 
         return collections, total
 
-    async def get_collection_crawls(self, coll_id: uuid.UUID, oid: uuid.UUID):
+    async def get_collection_crawl_resources(
+        self, coll_id: uuid.UUID, org: Organization
+    ):
         """Find collection and get all crawl resources"""
 
-        coll = await self.get_collection(coll_id)
+        coll = await self.get_collection(coll_id, org)
         if not coll:
             raise HTTPException(status_code=404, detail="collection_not_found")
 
         all_files = []
 
-        for crawl_id in coll.crawlIds:
-            org = await self.orgs.get_org_by_id(oid)
+        crawl_ids = await self.crawls.get_crawls_in_collection(coll_id)
+        for crawl_id in crawl_ids:
             crawl = await self.crawls.get_crawl(crawl_id, org)
```

Comment:
We should get all of the crawls in a batch (with list_crawls?), or perhaps in a few paginated queries (not sure if it makes difference here). We have no choice but to return all of the crawl wacz files here so can return the files in a flat list, to conform with Multi WACZ.